20190519-WebAbstract/BDA_FS19_WebAbstract.tex

```diff
@@ -124,12 +151,20 @@ \section{Ausblick}
 	\includegraphics[keepaspectratio,width=2.7cm]{img/HSLU_Logo_Header}
 \end{textblock*}
 
-TEXT
+\section{Spezielle Herausforderungen}
+Die Herausforderungen in diesem Projekt lagen vor allem in der Wissensbeschaffung. Beide Teammitglieder waren nicht vertraut mit der Durchführung einer Machbarkeitsstudie und es musste daher zuerst eine Recherche durchgeführt werden, wie eine Machbarkeitsstudie durchgeführt werden soll.
+
+Weiter war die Beschaffung eines Budgets für die Anschaffung der Hardware eine spezielle Hürde. Keiner der identifizierten Hersteller konnten Leihgaben zur Verfügung stellen und sowohl die Hochschule wie auch der Kunde konnten keine grösseren Geldbeträge allozieren, sodass der präferierte Hersteller beachtet werden konnte. Es musste also vonseiten des Projektteams eine kreative Lösung gefunden werden.
+
+\section{Ausblick}
+Die Machbarkeitsstudie sieht vor, dass die erarbeitete Lösung in einer weiteren Bachelorarbeit durchgeführt wird. Weiter wird ein klarer Budgetbetrag vorgesehen, welcher vom Kunden erbracht werden muss, bevor die Arbeit durchgeführt werden kann. Namentlich ist dies die Anschaffung der Hardware vom vorgesehenen Hersteller.
+
+Wird die Lösung wie im Konzept ausgearbeitet entwickelt, so besitzt sie einen grossen Neuheitsgrad, da HF RFID Chips in einer spezialisierten Umgebung (ausschliesslich Bibliotheken und deren Lager) eingesetzt werden, und dort noch keine Fertiglösungen für ein solches System existieren.
```

Comment:
Achtung so wie es jetzt geschrieben wurde, gibt es alternativen. Der Neuheitsgrad besteht nur in der Tag-Dichte.